docs-source/content/samples/simple/elastic-stack/weblogic-domain/_index.md

```diff
@@ -0,0 +1,335 @@
+---
+title: "WebLogic Domain"
+date: 2019-02-23T17:32:31-05:00
+weight: 6
+description: "Sample for configuring fluentd to export domain logs to ELK."
+---
+
+
+#### Overview
+This page describes to how to configure a WebLogic domain to use fluentd to send log information to ELK.
+
+The general mechanism of how this works is:
+* fluentd runs as a separate container in the admin and managed server pods
+* The log files reside on a volume that is shared between the `weblogic-server` and `fluentd` containers
+* fluentd tails the domain logs files and exports to ELK
+* A ConfigMap contains the filter and format rules for exporting log records
+
+For sample purposes, this document will assume a domain with the following attributes are being configured:
+* Domain name is `bobs-bookstore`
+* Kubernetes namespace is `bob`
+* Kubernetes secret is `bobs-bookstore-weblogic-credentials`
+* Elastic Search Configuration
```

Comment:
It is "Elasticsearch" and "configuration" should not have capital

Reply:
done in next push